check-pr-description/action.yaml

```diff
@@ -35,6 +35,9 @@ outputs:
   test-enabled:
     description: whether the tests should run or not
     value: ${{ steps.check-string.outputs.test-enabled }}
+  persist:
```

Comment:
sorry to nitpick but could you document the use of `#persist` in the readme for this action? 🙏

Reply:
@rokroskar no problem - done